docs/EMBEDDING.md

```diff
@@ -61,6 +61,9 @@ query string. Currently the following options are available:
 * `resize` - How to resize the remote session if it is not the same size as
   the browser window. Can be one of `off`, `scale` and `remote`.
 
+* `show_dot` - If a dot cursor should be shown when the remote server provides
+  no local cursor, or provides a fully-transparent (invisible) cursor.
+
```

Comment:
`API.md` also needs to be updated.

And these additions should probably be in the same commit as adds the feature in the code.